components/MonogramAvatar/MonogramAvatar.jsx

```diff
@@ -8,24 +8,31 @@ import styles from './monogram_avatar.scss'
 
 import Avatar from '../Avatar/Avatar'
 
+type AvatarSize = "smaller" | "small" | "base" | "large" | "larger"
+
 type Props = {
-  url: string,
-  personName: string,
-  className: string,
   border: boolean,
+  className: string,
+  personName: string,
+  size: AvatarSize,
+  url: string,
 }
 
 type State = {
   showInitials: boolean
 }
 
-const initials = (name: string): string => (
-  map(name.split(/\s/), name => name[0])
-)
+const initials = function(name: string): string {
+  if (!name) {
+    return <i className="fa fa-user"></i>
+  }
+  return map(name.split(/\s/), name => name[0])
```

Comment:
Can we limit to 2 characters? The design breaks if more than 2.

We've had a problem in the past with super long names...or using this for installers.